libafl_frida/Cargo.toml

```diff
@@ -44,7 +44,13 @@ track_hit_feedbacks = ["libafl/track_hit_feedbacks"]
 auto-download = ["frida-gum-sys/auto-download", "frida-gum/auto-download"]
 
 [build-dependencies]
+anyhow = "1.0"
```

Comment:
I think we should stick to our `libafl_bolts::Error` struct in LibAFL (and potentially implement more `.into()` cases)